app/scripts/modules/core/pipeline/config/stages/createLoadBalancer/createLoadBalancerStage.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.core.pipeline.stage.createLoadBalancerStage', [
+  require('../../../../utils/lodash.js'),
+  require('../../../../cloudProvider/cloudProvider.registry.js'),
+])
+  .config(function(pipelineConfigProvider, settings) {
+
+    // Register this stage only if infrastructure stages are enabled in settings.js
+    if (settings.feature.infrastructureStages) {
+      pipelineConfigProvider.registerStage({
+        key: 'upsertLoadBalancers',
+        label: 'Create Load Balancers',
+        description: 'Creates one or more load balancers',
```

Comment:
Is it worth explaining that this is an upsert operation in the description or label? i.e., if the load balancer exists, this stage will not blow up? Or that it will blow up, if that's the case. I don't know.